pkg/protocol/http1/ext/common.go

```diff
@@ -43,21 +43,27 @@ package ext
 
 import (
 	"bytes"
+	"errors"
 	"fmt"
 	"io"
+	"strings"
 
 	"github.com/cloudwego/hertz/internal/bytesconv"
 	"github.com/cloudwego/hertz/internal/bytestr"
 	errs "github.com/cloudwego/hertz/pkg/common/errors"
 	"github.com/cloudwego/hertz/pkg/common/utils"
 	"github.com/cloudwego/hertz/pkg/network"
+	"github.com/cloudwego/hertz/pkg/protocol"
 	"github.com/cloudwego/hertz/pkg/protocol/consts"
 )
 
 const maxContentLengthInStream = 8 * 1024
 
 var errBrokenChunk = errs.NewPublic("cannot find crlf at the end of chunk").SetMeta("when read body chunk")
 
+// Deprecated: Use github.com/cloudwego/hertz/pkg/protocol.NoBody instead.
+var NoBody = protocol.NoBody
```

Comment:
What's this for?

Reply:
Http2 also uses NoBody, but it's in the http1 package. If use it directly, it will cause cyclic dependence. So move it to protocol package. Here is a reference to avoid break change.

Reply:
Maybe put it in stream.go - as there is no reference for it in this file?